tests/system/fim/common_tasks/verify_alerts_json.py

```diff
@@ -105,16 +105,16 @@ def main():
                             help="Output path for missing files alerts.",
                             default="debug_missing_file_alerts.log")
 
-        parser.add_argument("-t", "--timeout", type=int, required=False, dest='time_gap',
-                            help="time gap between alerts.json alerts acquisitions. default: 60s",
-                            default="60")
+        parser.add_argument("-s", "--sleep", type=int, required=False, dest='sleep_time',
+          help="Sleep time between retries", default="60"
+        )
 
         parser.add_argument("-r", "--retry", type=int, required=False, dest='retry_count',
                             help="reading attempts on stopped alerts. default: 4 attemps",
                             default="4")
-        parser.add_argument("-w", "--whodata", type=bool, required=False, dest='whodata_check',
```

Comment:
Related to https://github.com/wazuh/wazuh-qa/pull/549/files#r384851561

Reply:
Same answer

Reply:
Same response, check it please